easybuild/easyconfigs/r/RAxML/RAxML-8.2.4-intel-2015b-hybrid-avx.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+
+name = 'RAxML'
+version = '8.2.4'
+versionsuffix ='-hybrid-avx'
```

Comment:
RAxML provides separate Makefiles for each of these, with preprocessor directives like `-D_AVX`, most likely because it contains assembly code, so it's not as simple as using `-march=native` or `-xHost`, you still need to use the right Makefile.